testdata/cheats/Etch.t.sol

```diff
@@ -14,12 +14,12 @@ contract EtchTest is DSTest {
         assertEq(string(code), string(target.code));
     }
 
-    function testEtchNotAvailableOnPrecompiles() public {
-        address target = address(1);
-        bytes memory code = hex"1010";
-        vm.expectRevert(
-            bytes("Etch cannot be used on precompile addresses (N < 10). Please use an address bigger than 10 instead")
-        );
-        vm.etch(target, code);
-    }
+    // function testEtchNotAvailableOnPrecompiles() public {
```

Comment:
What's up

Reply:
Ah, commented by mistake when making the PR. fix https://github.com/foundry-rs/foundry/pull/5223/commits/1e4f206afd4cb2bc7b301ef4bba7bf2b666fb331